code/game/machinery/computer/camera_advanced.dm

```diff
@@ -20,71 +20,94 @@
 	jump_action.Grant(user)
 
 /obj/machinery/computer/camera_advanced/check_eye(mob/user)
-	if (get_dist(user, src) > 1 || user.eye_blind)
-		if(user == current_user)
-			off_action.Activate()
-		return 0
-	return 1
+	if( (stat & (NOPOWER|BROKEN)) || !Adjacent(user) || user.eye_blind || user.incapacitated() )
```

Comment:
You can no longer use the advanced camera console if you're away from it or incapacitated, or if the console is broken or unpowered.